tgui/packages/tgui/interfaces/CrewConsole.js

```diff
@@ -177,7 +189,7 @@ const CrewTableEntry = (props, context) => {
         )}
       </Table.Cell>
       <Table.Cell>
-        {area !== undefined ? area : <Icon name="question" color="#ffffff" size={1} />}
+        {area !== undefined ? area : <Icon name="question" color="#ffffff" size={1} /> } {/* SKYRAT EDIT - Icon from text 'N/A*/}
```

Comment:
Just fixing the comment a bit. Also, I feel like the question mark looks off compared to everything else, no?
```suggestion
        {area !== undefined ? area : <Icon name="question" color="#ffffff" size={1} /> } {/* SKYRAT EDIT - Icon from text 'N/A'*/}
```

Reply:
Now would be the time to change it, what do you think?